docs/guides/routing.rst

```diff
@@ -189,13 +189,73 @@ Route middleware can be used to prevent access to a certain route, or to perform
 the route is rendered. Middleware can be used, e.g. to implement a paywall, or to log analytics,
 or to set open graph metatags.
 
-Elgg core implements several middleware handlers, including:
+Elgg core implements several middleware handlers. The following middleware can be found in the namespace ``\Elgg\Router\Middleware``:
 
- * ``\Elgg\Router\Middleware\Gatekeeper`` - prevent access by non-authenticated users
- * ``\Elgg\Router\Middleware\AdminGatekeeper`` - prevent access by non-admin users
- * ``\Elgg\Router\Middleware\AjaxGatekeeper`` - prevent access with non-xhr requests
- * ``\Elgg\Router\Middleware\CsrfFirewall`` - prevent access without CSRF tokens
- * ``\Elgg\Router\Middleware\SignedRequestGatekeeper`` - prevent access if the url has been tampered with
+Gatekeeper
+~~~~~~~~~~
+
+This gatekeeper will prevent access by non-authenticated users.
+
+AdminGatekeeper
+~~~~~~~~~~~~~~~
+
+This gatekeeper will prevent access by non-admin users.
+
+LoggedOutGatekeeper
+~~~~~~~~~~~~~~~~~~~
+
+This gatekeeper will prevent access by authenticated users.
+
+AjaxGatekeeper
+~~~~~~~~~~~~~~
+
+This gatekeeper will prevent access with non-xhr requests.
+
+PageOwnerCanEditGatekeeper
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This gatekeeper will prevent access if there is no page owner detected or if the page owner can't be editted.
```

Comment:
Incorrect. If no page owner is detected nothing happens, but if there is a page owner you need to be able to edit it